web/src/styles/scss/tables.scss

```diff
@@ -1,33 +1,70 @@
 @import '@cmsgov/design-system-support/src/settings/index';
 
-.table-cms {
+.budget-table {
   font-size: 12px;
-  &.table-cms__totals {
-    width: 42%; // roughly equal to full tables' cells; temp fix before switching font to courier
+  &.budget-table--totals {
+    width: 50%;
   }
-  &.centered-headers th {
-    text-align: center;
+  + .budget-table {
+    margin-top: $spacer-2;
   }
+
   th {
-    background-color: #DAE9F7;
-    font-weight: normal;
+    font-weight: $font-semibold;
   }
-  td {
-    background-color: #F3F8FC;
-  }
-  th,
-  td {
-    border-left: 4px solid $color-white;
-    border-top: 4px solid $color-white;
-    &:first-child {
-      border-left: none;
-    }
 
+  th, tr {
+    &.budget-table--total,
+    &.budget-table--subtotal,
+    &.budget-table--sum {
+      font-weight: $font-bold;
+      th, td {
+        font-weight: $font-bold;
+      }
+    }
   }
-  + .table-cms:not(.table-frozen-data){
-    margin-top: $spacer-2;
+
+  thead {
```

Comment:
I ***ALWAYS*** misread this as `<thread>` and I have to stop and think what the heck the `thread` element is...  Probably something @edmullen invented.

Reply:
If only there were a single, defined, semantic way to hierarchically converse....